### PR TITLE
W-11933628-image-search-cv

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**
Adding screenshots, links in See Also section, and adjusting content.

### CODE REVIEW

**Review by justus-schach — DISMISSED** [review]

File: rpa-builder/6.0/modules/ROOT/pages/toolbox-checks-image-search.adoc [hunk 1]
```diff
@@ -1,181 +1,239 @@
...
 used for the _Image search_ (only applicable if _Load Optimizer_ is
-activated)
+activated).
 
 == Variables
 
-* *Find Coordinate* The X/Y coordinate of where the picture was found
+* *Find Coordinate*
++
+The X/Y coordinate of where the picture was found.
```
Comment (Crispy-Salesforce) [inline rpa-builder/6.0/modules/ROOT/pages/toolbox-checks-image-search.adoc:158]: Would it be better to say "X and Y" and "coordinates"? A Google search showed that the most common expression is "X and Y coordinates".

File: rpa-builder/6.0/modules/ROOT/pages/toolbox-checks-image-search.adoc [hunk 2]
```diff
@@ -1,181 +1,239 @@
...
+16 bits, the original _Pattern image_ is converted on the fly to
 match the current system's color depth. You can adjust this within the
-*Workflow Properties*.
-//link:#WorkflowProperties[_Workflow Properties_].
+Workflow Properties.
 
 == Properties
 
-* *Compare Precision* This is an additional setting for the search
+* *Compare Precision*
```
Comment (Crispy-Salesforce) [inline rpa-builder/6.0/modules/ROOT/pages/toolbox-checks-image-search.adoc:46]: This "Compare Precision" setting appears in capital letters and no non-capital letters. Is this a label, or should it be in non-capital letters?

File: rpa-builder/6.0/modules/ROOT/pages/toolbox-checks-image-search.adoc [hunk 3]
```diff
@@ -1,181 +1,239 @@
...
+* *Edge detection threshold*
++
+Threshold for the difference (distance)
 between colors that leads to the detection of an edge. A low threshold
 leads to many detected edges because even a small difference in a
 pixel's color value in comparison to its neighbors' is detected as an
 edge. Conversely, a high threshold leads to fewer edges in the _Pattern
 image_ the algorithm is searching for.
-* *Hide Mouse* If this option is enabled, the mouse pointer is moved to
+* *Hide Mouse*
```
Comment (Crispy-Salesforce) [inline rpa-builder/6.0/modules/ROOT/pages/toolbox-checks-image-search.adoc:62]: These bullets do not take capital letters so far. Is this a label?

File: rpa-builder/6.0/modules/ROOT/pages/toolbox-checks-image-search.adoc [hunk 4]
```diff
@@ -1,181 +1,239 @@
...
+(_Properties grid_ only): Original coordinates
 (upper left corner, lower right corner) of the rectangular area on the
-desktop enclosing the _Search area_
-* *Wait time [ms]* The pause between two subsequent __Image search__es
-(only used if _Wait until image is steady_ is activated)
-* *Wait until image is steady* This function is used to perform a second
+desktop enclosing the _Search area_.
+* *Wait time [ms]*
++
+The pause between two subsequent __Image search__es
```
Comment (Crispy-Salesforce) [inline rpa-builder/6.0/modules/ROOT/pages/toolbox-checks-image-search.adoc:124]: It seems weird to me that the "es" of "searches" is not in italics.

**Review by Crispy-Salesforce — APPROVED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**